sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/aad/configuration/properties/AadAuthenticationProperties.java

```diff
@@ -111,41 +111,41 @@ public class AadAuthenticationProperties implements InitializingBean {
     private int jwtSizeLimit = RemoteJWKSet.DEFAULT_HTTP_SIZE_LIMIT; /* bytes */
 
     /**
-     * The lifespan(duration) of the cached JWK set before it expires. The default value is `5m`.
+     * The lifespan(duration) of the cached JWK set before it expires.
      */
     private Duration jwkSetCacheLifespan = Duration.ofMinutes(5);
 
     /**
-     * The refresh time(duration) of the cached JWK set before it expires. The default value is `5m`.
+     * The refresh time(duration) of the cached JWK set before it expires.
      */
     private Duration jwkSetCacheRefreshTime = Duration.ofMinutes(5);
 
     /**
-     * The redirect uri after logout. For instance, 'http://localhost:8080/'.
+     * The redirect uri after logout. For instance, `http://localhost:8080/`.
      * See Microsoft doc about Redirect URI for more details: https://learn.microsoft.com/azure/active-directory/develop/security-best-practices-for-app-registration#redirect-uri
      */
     private String postLogoutRedirectUri;
 
     /**
-     * If true activates the stateless auth filter AADAppRoleStatelessAuthenticationFilter. The default is false which
-     * activates AADAuthenticationFilter.
+     * If true activates the stateless auth filter `AADAppRoleStatelessAuthenticationFilter`. The default is false, which
+     * activates `AADAuthenticationFilter`.
      */
     private Boolean sessionStateless = false;
 
     /**
      * The OAuth2 authorization clients, contains the authorization grant type, client authentication method and scope.
-     * The clients will be converted to OAuth2 ClientRegistration, the other ClientRegistration information(such as client id, client secret) inherits from the delegated OAuth2 login client 'azure'.
-     * For instance,'
-     * authorization-clients.webapi.authorization-grant-type=on_behalf_of,
-     * authorization-clients.webapi.client-authentication-method=client_secret_post,
-     * authorization-clients.webapi.scopes[0]={WEB_API_APP_ID_URL}/WebApi.ExampleScope1,
-     * authorization-clients.webapi.scopes[0]={WEB_API_APP_ID_URL}/WebApi.ExampleScope2
-     * '.
+     * The clients will be converted to OAuth2 `ClientRegistration`, the other `ClientRegistration` information(such as client id, client secret) inherits from the delegated OAuth2 login client `azure`.
```

Comment:
Can we implement this in the doc plugin, so we can replace any '' in the comment, to \`\`

Reply:
Now implement this in the doc plugin, so revert these changes about ''.